build_tools/cmake/test.sh

```diff
@@ -59,8 +58,6 @@ if [[ "${IREE_LLVMAOT_DISABLE?}" == 1 ]]; then
 fi
 if [[ "${IREE_CUDA_DISABLE?}" == 1 ]]; then
   label_exclude_args+=("^driver=cuda$")
-fi
-if [[ "${IREE_CUDA_RUNTIME_DISABLE?}" == 1 ]]; then
   label_exclude_args+=("^uses_cuda_runtime$")
```

Comment:
do we need both driver=cuda and uses_cuda_runtime? I think the driver=cuda should be enough?

Reply:
The problem is that some tests (like simple embedded) don't have a driver filed, that's why we had to add a tag to mark those and make sure they don't run on build bots without cuda support.

Reply:
you can add a driver=cuda just as easily as uses_cuda_runtime :)

Reply:
🤦